### PR TITLE
Remove mention of the GPS plugin

### LINKED ISSUE #6840: Move GPS Tools GPSbabel device configuration widget to new page in global settings dialog (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#44458
Author: @nyalldawson
QGIS version: 3.22

**Move GPS Tools GPSbabel device configuration widget to new page in global settings dialog**

### PR Description:
Instead of a custom settings page in the plugin, move the GPS babel configuration to the main settings dialog:

image

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by DelazJ**
and adjust description (fixes #6840 and finishes #6849)

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by DelazJ** [comment]
@nyalldawson may I request a review please?
Moreover I am removing this [section](https://docs.qgis.org/3.28/en/docs/user_manual/working_with_gps/plugins_gps.html#download-of-points-tracks-from-gps-units) and plan to remove [that one](https://docs.qgis.org/3.28/en/docs/user_manual/working_with_gps/live_GPS_tracking.html#using-gpsmap-60cs) and following ones in next PRs. Any objections?
Thanks.

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -1140,35 +1140,63 @@ https://doc.qt.io/archives/qt-5.9/qnetworkproxy.html#ProxyType-enum
...
+
+     * :guilabel:`Waypoint download` from the device
+     * :guilabel:`Waypoint upload` to the device
+     * :guilabel:`Route download` from the device
+     * :guilabel:`Route upload` to the device
+     * :guilabel:`Track download` from the device
+     * :guilabel:`Track upload` to the device
+
+     This will probably be a GPSBabel command,
+     but you can use any other command line program that can create a GPX file.
```
Comment (nyalldawson) [inline docs/user_manual/introduction/qgis_configuration.rst:1178]:
```suggestion
     While the commands are usually GPSBabel commands, you can also use any other command line program that can create a GPX file.
```

File: docs/user_manual/working_with_gps/plugins_gps.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
-.. _plugin_gps:
+.. _gps_data:
 
-GPS Plugin
-==========
+Introducing GPS Data
```
Comment (nyalldawson) [inline docs/user_manual/working_with_gps/plugins_gps.rst:3]:
```suggestion
Introducing GNSS/GPS Data
```

that might help users searching for the more generic "gnss" term discover this page. See also https://github.com/qgis/qgis4.0_api/issues/124

**Review by nyalldawson — APPROVED** [review]

**Comment by DelazJ** [comment]
Thanks.
How about my plans to remove this [section](https://docs.qgis.org/3.28/en/docs/user_manual/working_with_gps/plugins_gps.html#download-of-points-tracks-from-gps-units) and [that one](https://docs.qgis.org/3.28/en/docs/user_manual/working_with_gps/live_GPS_tracking.html#using-gpsmap-60cs) and following ones in next PRs? Any objections? I have no idea if still correct and necessary.

File: docs/user_manual/working_with_gps/plugins_gps.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+=====================
```
Comment (DelazJ) [inline docs/user_manual/working_with_gps/plugins_gps.rst:4]:
```suggestion
===========================
```

**Pull request merged**